starknet_py/abi/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .model import Abi
+from .parser import AbiParser, AbiParsingError
```

Comment:
Should relative imports be used here?

Reply:
They stayed relative because it's init file or?

Reply:
Yes. Actually most of our imports in init use relative imports, so I guess it also makes sense to keep this consistent.